tests/rules/no-invalid-at-rules.test.js

```diff
@@ -165,5 +202,67 @@ ruleTester.run("no-invalid-at-rules", rule, {
 				},
 			],
 		},
+		{
+			code: "@foobar { body { font-size: 12px } }",
+			languageOptions: {
+				customSyntax: {
+					atrules: {
+						foobar: {
+							prelude: "<url>",
+						},
+					},
+				},
+			},
+			errors: [
+				{
+					messageId: "missingPrelude",
+					data: { name: "foobar" },
+					line: 1,
+					column: 1,
+					endLine: 1,
+					endColumn: 8,
+				},
+			],
+		},
+		{
+			code: "a { @apply; }",
+			languageOptions: {
+				customSyntax: tailwindSyntax,
+			},
+			errors: [
+				{
+					messageId: "missingPrelude",
+					data: {
+						name: "apply",
+						prelude: "",
+						expected: "<ident>+",
+					},
+					line: 1,
+					column: 5,
+					endLine: 1,
+					endColumn: 11,
+				},
+			],
+		},
+		{
+			code: "@config;",
+			languageOptions: {
+				customSyntax: tailwindSyntax,
+			},
+			errors: [
+				{
+					messageId: "missingPrelude",
+					data: {
+						name: "config",
+						prelude: "",
+						expected: "<string>",
+					},
+					line: 1,
+					column: 1,
+					endLine: 1,
+					endColumn: 8,
+				},
+			],
+		},
```

Comment:
```suggestion
		{
			code: "@config;",
			languageOptions: {
				customSyntax: tailwindSyntax,
			},
			errors: [
				{
					messageId: "missingPrelude",
					data: {
						name: "config",
					},
					line: 1,
					column: 1,
					endLine: 1,
					endColumn: 8,
				},
			],
		},
		{
			code: "@config foo;",
			languageOptions: {
				customSyntax: tailwindSyntax,
			},
			errors: [
				{
					messageId: "invalidPrelude",
					data: {
						name: "config",
						prelude: "foo",
						expected: "<string>",
					},
					line: 1,
					column: 9,
					endLine: 1,
					endColumn: 12,
				},
			],
		},
```

Same here, only name is used with `messageId: "missingPrelude"`. If we want to check the syntax, we can add another test with `messageId: "invalidPrelude"`.